get_molecule_xyz/get_molecule_xyz.py

```diff
@@ -18,10 +20,19 @@ def cas_to_xyz(xyz_title:str,cas_number:str) -> None:
 
     Returns
     -------
-        None
+        str describing the success
         Creates an xyz file for a molecule if it can be found 
     """
-    pass
+    molecule = nistchempy.get_compound(cas_number)
+    if molecule != None:
+        molecule.get_mol3D()
+        if molecule.mol3D != None:
+            molecule = Chem.MolToXYZFile(xyz_title+".xyz")
+            return xyz_title+".xyz created"
+        else:
+            return xyz_title+".xyz failed\n\tno mol3d found"
+    else:
+        return xyz_title+".xyz failed\n\tcas not found"
 
 
 # ToBeImplemented
```

Comment:
consider adding defaults that use the essential information only - interpreting file names based on defaults in cas_to_xyz